addons/project/models/project.py

```diff
@@ -1382,7 +1382,10 @@ def action_assign_to_me(self):
     # If depth == 3, return children to third generation
     # If depth <= 0, return all children without depth limit
     def _get_all_subtasks(self, depth=0):
-        children = self.mapped('child_ids').filtered(lambda children: children.active)
+        if self.env.context.get('active_test', True) is False:
+            children = self.mapped('child_ids')
+        else:
+            children = self.mapped('child_ids').filtered(lambda children: children.active)
```

Comment:
I think this is useless as mapped('child_ids') has no more the context={'active_test': False} attribute so it will always return what is needed in regards with the context it is executed in.
```suggestion
        children = self.child_ids
```

Reply:
@IT-Ideas , Done, It was done to solve the issue stat button count at: https://github.com/odoo/odoo/commit/e0a7add01fe3d634e6024b75a0f29240a8581d4d but now stat button does not exist so no need to filter it.

Thanks

Reply:
Hello @mba-odoo my point is that if the key is in the context, no need to do this logic as it will be anyway taken into account in the .mapped('child_ids').
Here the logic is:
 - if active_test is set to False:
   -> then we take all child_ids
 - if the key was no set or is True:
   -> then filter on the active (but then we actually would only have the active ones this is where my point is)

Is this clearer? Sorry If my first explanation wasn't clear enough